designer/client/components/Flyout/Flyout.tsx

```diff
@@ -1,11 +1,15 @@
 import React, { useContext, useEffect, useLayoutEffect, useState } from "react";
 import FocusTrap from "focus-trap-react";
 import { FlyoutContext } from "../../context";
+import { DataContext } from "../../context";
 import { i18n } from "../../i18n";
 
 import "./Flyout.scss";
+import { bool } from "aws-sdk/clients/signer";
 
-export function useFlyoutEffect(props = {}) {
+type Props = {};
+
+export function useFlyoutEffect(props: Props) {
```

Comment:
was there a type complaint here? 
`type Props = {}` is not equivalent to `fn(props = {})` 

when declaring parameters for a function, eg, `function useFlyoutEffect(props)`, you can set the default of a parameter of props with `= ...` 

If you declare `function useFlyoutEffect(props = true)`, then call it without parameters `useFlyoutEffect()`, `console.log(props)` will be `true`.

These are equivalent 
```
type Props = {};

//same as 
function useFlyoutEffect(props: {})
```
which is to say, you must only pass in an empty object, which isn't right. 

I want it to default to an empty object, since we try to access the property .show (`props.show`). If you try to access a property of an undefined value eg `(undefined).show` it will throw an error/crash